### PR TITLE
MOB-696: Update version number to v2.0.0

### PR DESCRIPTION
**Pull request by matir91**
## Description
- Update readme 
- Version bump

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,107 +1,163 @@
...
+The ID.me WebVerify SDK for Android is a library that allows you to verify a user's group affiliation status using ID.me's platform. A sample project has been provided to delineate the integration process.
 
-#Release Information 
-- **SDK Version:** 1.2.0 (August 11, 2016)
+## Specification
+### Supported Android Versions
+
+Android API 16 (Jellybean) and above is supported.
+
+## Release Information
```
Comment (m-revetria) [inline README.md]: What do you think about adding a link to the changelog here? https://github.com/IDme/ID.me-WebVerify-SDK-iOS/pull/34/files#diff-04c6e90faac2675aa89e2176d2eec7d8R11

File: README.md [hunk 2]
```diff
@@ -1,107 +1,163 @@
...
+repositories {
+    maven {
+        url 'https://oss.sonatype.org/content/groups/public'
+    }
 }
+
 ```
 
 ### Manual
 Copy the `.aar` file into your project's `libs` folder.
```
Comment (m-revetria) [inline README.md]: which `.aar` file? from where can I copy it?

File: README.md [hunk 3]
```diff
@@ -1,107 +1,163 @@
...
-### Sample Project
-To run the sample project you'll need to replace the `clientID` and `redirectUri` values in `MainActivity.java` with those values that are stored withing your registered app at [http://developer.id.me](http://developer.id.me).
-
-## Execution
-Both the OAuth flow and verification occur through a WebView that initialized from withn the SDK. Upon successful completion the activity will close and the JSON string will be sent back to the calling activity.
-
-To start the `WebViewActivity` you must initialize a `IDmeWebVerify` object,
+## Setup
+### Step 1
+#### Using gradle
```
Comment (m-revetria) [inline README.md]: as the only documented way is using gradle I would remove this title?

File: README.md [hunk 4]
```diff
@@ -1,107 +1,163 @@
...
 
-The params in the constructor are as follows:
 
-- `clientID`: The clientID provided by ID.me when registering the app at [http://developer.id.me](http://developer.id.me).
-- `redirectURI`: The redirectURI provided to ID.me when registering your app at [http://developer.id.me](http://developer.id.me)
-- `affiliationType`: The type of group verfication that should be presented.
-- `activity`: The activity that is creating the Class Object.
-- `returnProperties`: Indicates returning the user properties or an access token which you can use to get user properties.
+## Execution
+Both the OAuth flow and verification occur through a device browser that initialized from within the SDK.
```
Comment (m-revetria) [inline README.md]: I think it should say *(...) through a device browser that **is** initialized (...)*

File: README.md [hunk 5]
```diff
@@ -1,107 +1,163 @@
...
+        idmeAuthRedirectScheme: "CUSTOM_SCHEME"
+    ]
+    ```
+- Replace the app scope in the [Scope class](me.id.webverify/app/src/main/java/me/id/meidwebverify/Scope.java) with the one defined for the application. 
+## Results and Error Handling
+All results will be passed in the functions callbacks.
 
 ## Internet Connectivity
-Internet connectivity is required, as the verificaiton occurs through a webView.
+Internet connectivity is required, as the verificaiton occurs through the device browser.
```
Comment (m-revetria) [inline README.md]: misspelling: **verification**

Comment (m-revetria) [inline README.md]: Maybe you can link the RFC to give more context on why this was changed

File: README.md [hunk 6]
```diff
@@ -1,107 +1,163 @@
...
+- `redirectUri`: The redirectUri provided to ID.me when registering your app at [http://developer.id.me](http://developer.id.me).
 
-The params in the constructor are as follows:
 
-- `clientID`: The clientID provided by ID.me when registering the app at [http://developer.id.me](http://developer.id.me).
-- `redirectURI`: The redirectURI provided to ID.me when registering your app at [http://developer.id.me](http://developer.id.me)
-- `affiliationType`: The type of group verfication that should be presented.
-- `activity`: The activity that is creating the Class Object.
-- `returnProperties`: Indicates returning the user properties or an access token which you can use to get user properties.
+## Execution
```
Comment (m-revetria) [inline README.md]: It sounds to me more like the **Usage** of the SDK, maybe you can move these points *Results and Error Handling* and *Internet Connectivity* to this section. What do you think?

File: README.md (hunk 3 above)
Comment (matir91) [inline README.md]: we should include the other way, but for now I can remove it

**Pull request merged**